# Shan Yu, Ph.D.

![Shan Yu](./Shan Yu.png)

## Shan Yu

329 Jinpeng Street, Sandun Town, Xihu District, Hangzhou, Zhejiang, China, 310030

Email: yushan2519@163.com

Tel: (+86) 156 1158 2166

### 1. Summary:

**Shan Yu** is a highly motivated bioinformatician with a strong interest in developing methods to uncover the cell identity and characterize the cell state transition from multiple perspectives. He is proficient in a variety of skills including **single-cell RNA-seq analysis**, **data visualization**, and **R package development**. For the past 5 years, he mainly focused on exploring the cellular heterogeneity of hematopoietic stem cells (2017 *Nat Commun*, 2018 *Autophagy*, 2021 *Cell Stem Cell*, and 2022 *Sci Adv*) using bioinformatic methods. Most recently, he is looking for a postdoctoral position in computational biology which focuses on developing single-cell multi-omics analysis methods.

### 2. Education:

2019 - 2022, Ph.D. in Biochemistry and Molecular Biology, Peking Union Medical College, Beijing, China

2016 - 2019, Master in Biochemistry and Molecular Biology, Peking Union Medical College, Beijing, China

2012 - 2016, Bachelor of Biological Sciences, Hainan University, Haikou, China

### 3. Skills:

1. Strong programming skills in using R language for statistical analysis, visualization, and software development.
2. Proficient in high-throughput sequencing data (including bulk & single-cell RNA-seq, ChIP-seq, CLIP-seq, and single-cell Nanopore-seq) analysis.
3. Extensive collaboration experiences with biologists.
4. Fluent in English writing and speaking.

### 4. Academic experience:

Here list of the main studies for the last 5 years:

#### I. A computational framework for transcription factor activity analysis (submitted to ***Nature Cell Biology***).

This research developed an integrated computational framework for gene regulatory network inference and transcription factor activity estimation from single-cell RNA sequencing data. In brief, we utilized an entropy-based algorithm to infer the weighted regulatory relationship between regulators and targets by decomposing the unique information from multivariate information. This framework Embed several functions such as transcriptional factor activity quantification, dimensionality reduction, cell-type-specific regulon analysis, pathway enrichment analysis, and visualization. This work aims to establish a comprehensive single-cell analysis framework for transcription factors just like other commonly used single-cell pipelines such as Seurat.

\* First author, mainly responsible for **study design**, **software development**, **bioinformatics analysis**, **visualization** and **manuscript writing**. Softwares from this project were deposited in GitHub: [https://github.com/hmutpw/scATFR](https://github.com/hmutpw/scATFR) and [https://github.com/hmutpw/PUIC](https://github.com/hmutpw/PUIC/).

#### II. Constructing the RNA subcellular localization landscape of embryonic stem cells (2023 ***Stem Cell Reports***).

This research comprehensively characterized the subcellular localization pattern of RNA molecules in embryonic stem cells. We identified several genes that showed distinct localization patterns among five different subcellular locations. Meanwhile, we explored the subcellular localization pattern of alternative splicing events between cytoplasm and nucleus and predicted several RNA binding proteins that potentially regulate the nuclear export.

\* Co-first author, mainly responsible for **bioinformatics analysis**, **visualization**, and **manuscript writing**.

#### III. Single-cell sequencing reveals the critical role of alternative splicing in hematopoietic stem cell formation (2022 ***Sci Adv***)

This research constructed an isoform-based transcriptional atlas of the murine endothelial-to-hematopoietic stem cell using single-cell full-length RNA sequencing data. In brief, we comprehensively described the isoform diversity and alternative splicing landscape during hematopoietic stem cell formation. Then, we also identified hundreds of stage-specific isoforms that potentially affect the hematopoietic stem cell formation and validated their expression using single-cell Nanopore sequencing of full-length transcripts. Subsequently, in vitro knockdown experiments showed that knockdown of some hemogenic-specific isoforms indeed affects the generation of  HSCs. Next, motif enrichment combined with knockout experiments confirmed that the RNA-binding protein (RBP), Srsf2, could affect the specialization of endothelial cells into hematopoietic stem cells by regulating the AS of key transcription factors such as Runx1 and Myb. 

![SA Abstract](./SA_figure.png)

\* Co-first author, mainly responsible for **study design**, **bioinformatics analysis**, **visualization**, and **manuscript writing**. The code was deposited in GitHub:[https://github.com/hmutpw/scAS/](https://github.com/hmutpw/scAS/).

#### IV. *hnRNPLL* controls pluripotency exit of embryonic stem cells by modulating alternative splicing of *Tbx3* and *Bptf* (2021 ***EMBO J***)

This research revealed that RNA binding protein *hnRNPLL* regulates the embryonic stem cell exit from pluripotency by modulating the alternative splicing of some key transcription factors that maintain the pluripotency.


![EMBOJ Abstract](./EMBOJ_figure.png)

\* Co-first author, mainly responsible for **bioinformatics analysis**, and **visualization**.

#### V. Depicting the dynamic activity of autophagy at the transcriptome level during the human stem cell differentiation (2018 & 2019, ***Autophagy***)

This research investigated the dynamic patterns of autophagy-related genes during human fetal digestive tract development using single-cell RNA sequencing data. Meanwhile, we also identified several autophagy-related genes that potentially regulate monocytic-granulocytic differentiation.

![Autophagy Abstract](./Autophagy_figure.png)

\* First author, mainly responsible for **bioinformatics analysis**, **visualization**, and **manuscript writing**.

#### VI. Construction of the RNA subcellular localization resources. (2017 & 2021, ***Nucleic Acids Res***)

These studies constructed and updated the RNA subcellular localization database for the transcriptome level.

![NAR Abstract](./NAR_figure.png)

\* Co-first author, mainly responsible for **data collecting** and **visualization**.

### 5. Research Interests:

Single-cell multi-omics

Machine learning

Software development

Hematopoietic stem cell

### 6. Full list of Publications:

( †: equally contribution; \*: corresponding author )

1. Xinjie Bao†, Gengchao Wang†, **Shan Yu†**, Jian Sun, Liu He, Hualu Zhao, Yanni Ma, Fang Wang, Xiaoshuang Wang, Renzhi Wang\*, Jia Yu\*. Transcriptomic analysis identifies a tumor subtype mRNA classifier for invasive non-functioning pituitary neuroendocrine tumor diagnostics. ***Theranostics***. 2021 Jan 1;11(1):132-146.[Full-text](https://www.thno.org/v11p0132.htm)

2.	Pengzhong Su, Jiahuan He, **Shan Yu**, Xiaoshuang Wang, Jia Yu\*. Expression and function of Met-tRNAiMet carrier proteins in acute myeloid leukemia. ***Basic & Clinical Medicine***. 2021; 41(4): 501-507.[Full-text](http://journal11.magtechjournal.com/Jwk_jcyxylc/EN/abstract/abstract13844.shtml)

3.	**Shan Yu**, Xiao-Shuang Wang, Kai-Can Cao, Xin-Jie Bao, Jia Yu\*. Identification of CDK6 and RHOU in Serum Exosome as Biomarkers for the Invasiveness of Non-functioning Pituitary Adenoma. ***Chin Med Sci J***. 2019 Sep 30;34(3):168-176.[Full-text](http://cmsj.cams.cn/EN/10.24920/003585)

4.	Zhengyi Zhang†, Jingnan Pi†, Dongling Zou†, Xiaoshuang Wang†, Jiayue Xu†, **Shan Yu**, Ting Zhang, Feng Li, Xianxie Zhang, Hualu Zhao, Fang Wang, Dong Wang\*, Yanni Ma\*, Jia Yu\*. microRNA arm-imbalance in part from complementary targets mediated decay promotes gastric cancer progression. ***Nat Commun***, 2019; 10(1): p. 4397.[Full-text](https://www.nature.com/articles/s41467-019-12292-5)

5.	Xiaoshuang Wang, **Shan Yu**, Jinrong He, Jia Yu\*. Function of RNA binding protein KSRP in type M5 acute myeloid leukemia cells. ***Basic & Clinical Medicine***. 2018; 38(6): 821-826. [Full-text](http://journal11.magtechjournal.com/Jwk_jcyxylc/CN/Y2018/V38/I6/821)

6. Xiaoshuang Wang, Jinrong He, **Shan Yu**, Jia Yu\*. Methyltransferase-like 3 Promotes the Proliferation of Acute Myeloid Leukemia Cells by Regulating N6-methyladenosine Levels of MYC. ***Acta Academiae Medicinae Sinicae***. 2018; 40(6): 308-314.[Full-text](http://journal13.magtechjournal.com/yxkxy/CN/10.3881/j.issn.1000-503X.2018.03.002)
    
7. Hongmei Zhao, Xiaoshuang Wang, Ping Yi, Yanmin Si, Puwen Tan, Jinrong He, **Shan Yu**, Yue Ren, Yanni Ma, Junwu Zhang, Dong Wang\*, Fang Wang\*, Jia Yu\*. KSRP specifies monocytic and granulocytic differentiation through regulating miR-129 biogenesis and *RUNX1* expression. ***Nat Commun***. 2017 Nov 10;8(1):1428. [Full-text](https://www.nature.com/articles/s41467-017-01425-3)


### 7. Other materials:
GitHub: [https://github.com/ShanYu2519](https://github.com/ShanYu2519)

Academic transcripts: [English version](./transcript_English.jpg)    [中文成绩单](./transcript_Chinese.jpg)

